x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -1162,13 +1162,23 @@ class PackagePolicyClientImpl implements PackagePolicyClient {
         ...new Set(result.filter((r) => r.success && r.policy_id).map((r) => r.policy_id!)),
       ];
 
+      const agentPoliciesWithEndpointPackagePolicies = [
```

Comment:
This could be a `Set`, so no array transformation is needed. Then below you can use the `.has` method from `Set` instead of array `inculdes`